docs/source/topics/proc_getting-shell-access-to-the-openshift-cluster.adoc

```diff
@@ -1,17 +1,21 @@
 [id="getting-shell-access-to-the-openshift-cluster_{context}"]
 = Getting shell access to the OpenShift cluster
 
-Direct access to the OpenShift cluster is not needed for regular use and is strongly discouraged.
 To access the cluster for troubleshooting or debugging purposes, follow this procedure.
 
+[NOTE]
+====
+Direct access to the OpenShift cluster is not needed for regular use and is strongly discouraged.
```

Comment:
It might be better to explain this oin future move as "direct access to the cluster" means logging in to the VM ?